engine/src/configuration/tests/python_tests.cpp

```diff
@@ -1,9 +1,8 @@
 /*
  * python_tests.cpp
  *
- * Copyright (c) 2001-2002 Daniel Horn
```

Comment:
I'd really like us to adopt some standard copyright text for all files. This should be something like:
* Created by Daniel R. Horn.
* Original dev team... 
* Current dev team...
* Copyright (c) 2001-2025 by the above develoeprs

I have no idea if this is valid. I just think that we've sort of erased the original work of the prior dev team and sometimes Daniel as well.
*

Reply:
We should absolutely have a standard copyright text. Something like what you're suggesting might work, although it's a little more verbose than what I've been using.

Let's discuss this further, maybe in a Discussion under the GitHub org.

Reply:
Started a discussion here: https://github.com/orgs/vegastrike/discussions/968